### DISCUSSION TITLE
AddToThis replcaing the share buttons

### DISCUSSION THREAD

**Discussion by monksy** [daattali/beautiful-jekyll#1283] [General]
I was asking for enhancements for the share buttons, but this might look like a better way to handle that situation: 

https://www.addtoany.com/

Thoughts?

**Comment by daattali** [comment]
It's a good idea, and I do like their lightweight design.
I wish I knew about them 8 years ago. I'm not sure how to handle this change in the config, this should not warrant a breaking change, but it doesn't really work well with the current config parameteres. Adding a brand new feature is easy becaue a new config parameter gets added and for old websites we can use a default value, but changing the way an existing feature works is very tricky. Let me know if you have a good solution for this.
